src/workbench/SideBar.tsx

```diff
@@ -161,62 +157,27 @@ const SideBarContents = ({ panes, ...props }: SideBarContentsProps) => {
         index={index}
       >
         <TabList>
-          <Box
-            width="3.75rem"
-            mt="1.2rem"
-            ml="auto"
-            mr="auto"
-            mb="max(11.5vh, 7.7rem)"
-          ></Box>
-          {panes.map((p, i) => (
-            <Tab
-              key={p.id}
-              color="white"
-              height={width}
-              width={width}
-              p={0}
-              position="relative"
-              className="sidebar-tab" // Used for custom outline below
-              onClick={handleTabClick}
-            >
-              <VStack spacing={0}>
-                {i === index && (
-                  <Corner
-                    id="bottom"
-                    position="absolute"
-                    bottom={-cornerSize + "px"}
-                    right={0}
-                  />
-                )}
-                {i === index && (
-                  <Corner
-                    id="top"
-                    position="absolute"
-                    top={-cornerSize + "px"}
-                    right={0}
-                    transform="rotate(90deg)"
-                  />
-                )}
-                <VStack spacing={1}>
-                  <Icon boxSize={6} as={p.icon} mt="3px" />
-                  <Text
-                    m={0}
-                    fontSize={13}
-                    borderBottom="3px solid transparent"
-                    sx={{
-                      ".sidebar-tab:focus &": {
-                        // To match the focus outline
-                        borderBottom: "3px solid rgba(66, 153, 225, 0.6)",
-                      },
-                    }}
-                  >
-                    {p.title}
-                  </Text>
-                </VStack>
-              </VStack>
-            </Tab>
-          ))}
-          <VStack mt="auto" mb={1} spacing={0.5} color="white">
+          <Box width="3.75rem" flex={1} maxHeight="8.9rem" minHeight={8}></Box>
+          <CustomTab
+            color="gray.50"
+            handleTabClick={handleTabClick}
+            activeTabId={tab ? tab : panes[0].id}
```

Comment:
`tab` is undefined by default. This feels like a fudge to have the 'explore' tab the default tab on page load after the refactor. Better way?